src/BBT.StructureTools/Copy/ICopyChildInterception.cs

```diff
@@ -3,14 +3,14 @@
     /// <summary>
     /// Interface to intercept the copy process.
     /// </summary>
-    /// <typeparam name="TClassToCopy">Type of to copied class.</typeparam>
+    /// <typeparam name="TClassToCopy">Type of the copied class.</typeparam>
     public interface ICopyChildInterception<in TClassToCopy> : IBaseAdditionalProcessing
         where TClassToCopy : class
     {
         /// <summary>
-        /// This method will called before the copy process of the model in the type parameter starts.
+        /// This method will be called before the copy process of the model in the type parameter starts.
         /// </summary>
         /// <returns><code>True</code> if the model must not copy, otherwise. <code>False</code></returns>
-        bool CalcopyChild(TClassToCopy source, TClassToCopy target);
+        bool BeforeCopyingChild(TClassToCopy source, TClassToCopy target);
```

Comment:
Not sure if this is the correct name here

Reply:
`ShallCopyChild` or something would probably be a better name.

Reply:
To add a bit of context, Interceptions are a way of controlling the copy process dynamically based on the actual data, and either continue or stop at a given "branch" of the tree.